pkg/daemon/ceph/osd/nsenter.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package osd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+)
+
+const (
+	// mountNsPath is the default mount namespace of the host
+	mountNsPath = "/rootfs/proc/1/ns/mnt"
```

Comment:
Is there any risk this could be different on downstream builds?

Reply:
`rootfs` always gets mounted at the same location AFAIK.